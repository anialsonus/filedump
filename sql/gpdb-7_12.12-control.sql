-- control file
\! pg_filedump -c test/pg_control.gp7_12.12
