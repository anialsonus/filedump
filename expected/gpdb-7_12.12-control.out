-- control file
\! pg_filedump -c test/pg_control.gp7_12.12

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: test/pg_control.gp7_12.12
* Options used: -c
*******************************************************************

<pg_control Contents> *********************************************

                          CRC: Correct
           pg_control Version: 12010700
              Catalog Version: 302307241
            System Identifier: 7285811498409084924
                        State: IN PRODUCTION
       Last Checkpoint Record: Log File (0) Offset (0x08000060)
  Last Checkpoint Record Redo: Log File (0) Offset (0x08000028)
          |-       TimeLineID: 1
          |-         Next XID: 0/1076
          |-         Next OID: 24576
          |- Next Relfilenode: 12502
          |-       Next Multi: 1
          |-    Next MultiOff: 0
       Minimum Recovery Point: Log File (0) Offset (0x00000000)
          Backup Start Record: Log File (0) Offset (0x00000000)
            Backup End Record: Log File (0) Offset (0x00000000)
End-of-Backup Record Required: no
       Maximum Data Alignment: 8
        Floating-Point Sample: 1234567
          Database Block Size: 32768
           Blocks Per Segment: 32768
              XLOG Block Size: 32768
            XLOG Segment Size: 67108864
    Maximum Identifier Length: 64
           Maximum Index Keys: 32
             TOAST Chunk Size: 8140

